### DISCUSSION TITLE
Possible to train once and then add totally different person later to the same model?

### DISCUSSION THREAD

**Discussion by jysung** [TheLastBen/fast-stable-diffusion#354] [Q&A]
Is it possible to 
1. train first on 30 images of SubjectA for 3000 iterations and save the checkpoint file
2. load colab with that checkpoint file, and then train a new session with only 10 images of SubjectB images for another 1000 iterations?

**Comment by TheLastBen** [comment]
Yes you can, I recommend setting the text encoder to 30% and the steps to +3000, 4000 is a sweet spot
